curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -328,6 +330,16 @@ bool DiskCacheManager::IsDiskCacheSafe() {
     return false;
 }
 
+bool DiskCacheManager::IsExceedFileNums() {
+    uint64_t fileNums = cachedObjName_->Size();
+    if (fileNums >= maxFileNums_) {
+        VLOG(6) << "file nums is: " << fileNums
```

Comment:
This log is printed once as the error level, which can be alarmed. The specific value can be viewed through metric

Reply:
cachedObjName_ is an LRU, why not do constraints through LRU?

Reply:
todo: https://github.com/opencurve/curve/issues/1534